.github/workflows/autoformat.yml

```diff
@@ -50,6 +56,8 @@ jobs:
     timeout-minutes: 30
     needs: check-comment
     if: ${{ needs.check-comment.outputs.should_autoformat == 'true' }}
+    permissions:
+      pull-requests: write  # push autoformat changes to PR
```

Comment:
Do we need this? I think commit push is performed with `MLFLOW_AUTOMATION_TOKEN`.

Reply:
Ah yeah, you're right, I missed that. I'll fix that now.

It will need `pull-requests: read` though, since the `GITHUB_TOKEN` is used in the `diff` step, which looks at the files modified in the PR.